src/main.rs

```diff
@@ -113,6 +113,10 @@ impl Kopium {
                         if scope == "Namespaced" {
                             println!(r#"#[kube(namespaced)]"#);
                         }
+
+                        if crd.status.is_some() {
```

Comment:
this needs to be something like:

```suggestion
                        if let Some(subres) = version.subresources.status {
                            if subres.status.is_some() {
```

Reply:
the info comes from the version struct: https://arnavion.github.io/k8s-openapi/v0.14.x/k8s_openapi/apiextensions_apiserver/pkg/apis/apiextensions/v1/struct.CustomResourceDefinitionVersion.html

Reply:
Changed a bit your suggestion, thanks for the pointer (quite new to the various libraries here)